components/base/Typography.tsx

```diff
@@ -0,0 +1,151 @@
+import * as React from "react";
+import styled from "@emotion/styled";
+import classnames from "classnames";
+import { POPPINS, ROBOTO } from "../typography";
+import {
+  HEADLINE_TEXT,
+  PARAGRAPH_TEXT,
+  WHITE_PRIMARY,
+  GREY_5,
+  GREY_4,
+} from "../colors";
+
+// Typography function
+function Typography({ text, styleVariant, tag, isDark }: TypographyProps) {
+  return (
+    <TypographyStyles
+      as={`${tag}`}
+      className={classnames(`${styleVariant}`, { dark: isDark })}
+    >
+      {text}
+    </TypographyStyles>
+  );
+}
+
+// Typography prop types
+type TypographyProps = {
+  text: string;
```

Comment:
Let's change this from `text` to `children` to be consistent with the built-in components (`h1`, `h2`, `p` etc.).

That way we can add these components as-is to `MDX_COMPONENTS` since MDX will pass the text as the `children` prop.